inscriptions/ISic003523.xml

```diff
@@ -207,12 +207,12 @@
         <body>
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#DEE">
                 <ab>
-                    <lb n="1"/><g ref="#ramus"/> Dis Manibus <g ref="#ramus"/>
+                    <lb n="1"/><g ref="palm">⸙</g> Dis Manibus <g ref="palm">⸙</g>
```

Comment:
missing the hashtag, to show that it picks up an id in the chardecl.xml file (apologies if I missed it out in my example!), ie. ref="#palm"